index.js

```diff
@@ -3778,6 +3778,24 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# fromPairs :: Array (Pair String a) -> StrMap a
+  //.
+  //. Returns a new string map formed by a list of key-value pairs.
```

Comment:
I think this is a slight improvement:

_Returns a string map containing the key–value pairs specified by the given array._

Note: Please use an [en dash][1] in _key–value pairs_. :)


[1]: http://www.thepunctuationguide.com/en-dash.html